setup.py

```diff
@@ -18,7 +18,10 @@
       author='Matheus Cardoso',
       author_email='matheus.mcas@gmail.com',
       license='MIT',
+      install_requires=[
+          'six==1.10.0'
```

Comment:
This I've done that because the requirements is using this version, I don't know if newer version would be compatible. I'm going to do this test as soon as possible and chanfe to ">="